class-instant-articles-post.php

```diff
@@ -759,17 +759,29 @@ public function add_ads_from_settings() {
 
 			case 'embed':
 				if ( ! empty( $settings_ads['embed_code'] ) ) {
+					$sectionAds = $settings_ads['embed_code'];
+					$tokenizeAds = explode("<!-- Embed Ad Block -->", $sectionAds);
```

Comment:
I'll make it simpler by using somthing like <!-- adblock -->, then normalize the token to a proper case. "Embed Ad Block" was starting to sound a little long to type :)